# Existing set of patches from the desktop gate
# Can be potentially sent upstream 

diff -ur iso-codes-3.21-orig/iso_3166/iso_3166.pot iso-codes-3.21/iso_3166/iso_3166.pot
--- iso-codes-3.21-orig/iso_3166/iso_3166.pot	Fri May 25 07:31:29 2012
+++ iso-codes-3.21/iso_3166/iso_3166.pot	Fri May 25 07:32:16 2012
@@ -1455,7 +1455,7 @@
 msgstr ""
 
 #. name for TWN, official_name for TWN
-msgid "Taiwan, Province of China"
+msgid "Taiwan"
 msgstr ""
 
 #. common_name for TWN
diff -ur iso-codes-3.21-orig/iso_3166/iso_3166.xml iso-codes-3.21/iso_3166/iso_3166.xml
--- iso-codes-3.21-orig/iso_3166/iso_3166.xml	Fri May 25 07:31:29 2012
+++ iso-codes-3.21/iso_3166/iso_3166.xml	Fri May 25 07:32:41 2012
@@ -1270,8 +1270,8 @@
 		alpha_3_code="TWN"
 		numeric_code="158"
 		common_name="Taiwan"
-		name="Taiwan, Province of China"
-		official_name="Taiwan, Province of China" />
+		name="Taiwan"
+		official_name="Taiwan" />
 	<iso_3166_entry
 		alpha_2_code="TJ"
 		alpha_3_code="TJK"
